scripts/patientSignUp.js

```diff
@@ -101,7 +101,7 @@ if(formSubmit()) {
 //     }
 //     if (i > 1) {  
 //         alert('Is Working');
-//         window.location = "https://hackathonv4.glitch.me/FirebaseRTC/public/patWaitingRoom.html";      
+//         window.location = "https://hackathonv4.glitch.me/public/templates/patWaitingRoom.html";      
```

Comment:
This is a comment but the url is wrong here. I think the reference needs to remain since the projecton glitch hasn't changed. 

Either delete this comment or don't change the URL here.